errorDisplay.js

```diff
@@ -3,108 +3,87 @@
 
 define(function (require, exports, module) {
 	"use strict";
-	
+
 	var EditorManager  = brackets.getModule("editor/EditorManager"),
 		ExtensionUtils = brackets.getModule("utils/ExtensionUtils"),
 		lineWidgetHTML = require("text!inlineWidget.html"),
-		widgetsErrors  = [],
-		gutters        = [];
+		currentErrorWidget;
 
 	ExtensionUtils.loadStyleSheet(module, "main.less");
 	require("tooltipsy.source");
-	
-	//Function that highlights the line(s) with errors
-	function markErrors(lineStart, lineEnd, charStart, charEnd) {
-		var editor   = EditorManager.getFocusedEditor();
-		var allMarks = editor._codeMirror.getAllMarks();
-
-		if(!allMarks.length) {
-			editor._codeMirror.markText({line: lineStart, ch: charStart},
-				{line: lineEnd, ch: charEnd},
-				{className: "errorHighlight"});
-		}
+
+	function initGutter() {
+		var errorGutter = document.createElement("div");
+		errorGutter.className = "errors";
```

Comment:
I'm confused about `errorGutter`.  You create the element here, set a `className`, then never use it again, nor add it to the DOM.  Why?